### DISCUSSION TITLE
Show Title and/or Artist on lock screen or via bluetooth

### DISCUSSION THREAD

**Discussion by wessel01** [madeofpendletonwool/PinePods#316] [Ideas]
Would it be difficult to show the episode title, artist and/or podcast title of the episode playing instead of only 'Pinepods'. That would really improve the experience when using pinepods when connected to my car display via bluetooth or when showing on the lock screen of my iphone.

Regards,

Wessel

**Comment by madeofpendletonwool** [comment]
Hi there @wessel01! Yes, to answer your question and it's already implemented and coming very soon! What you're looking for is better interactions with the browser apis to interact with the android and ios apis that show audio metadata which I implemented awhile back. Version 0.7.0 is going to include this and will be out in just a couple days. I'm putting on finishing touches currently! Keep an eye out!

**Reply by wessel01** [reply]
Thanks. Looking forward to it. Great job btw.
